wasp/src/main/java/com/orhanobut/wasp/WaspHttpStack.java

```diff
@@ -0,0 +1,16 @@
+package com.orhanobut.wasp;
+
+import com.android.volley.toolbox.HttpStack;
+
+import javax.net.ssl.SSLSocketFactory;
+
+/**
+ * @author Emmar Kardeslik
+ */
+public interface WaspHttpStack {
```

Comment:
What do you think if we return T instead of HttpStack

<code>
public interface WaspHttpStack< T> {
      public < T> T getHttpStack():
..
</code>

and use it in OkHttpStack like below
<code>
 class OkHttpStack extends HurlStack implements WaspHttpStack< HttpStack> {
</code>